python/sparkdl/transformers/param.py

```diff
@@ -92,11 +97,58 @@ def getOutputCol(self):
         return self.getOrDefault(self.outputCol)
 
 
-# New in sparkdl
-
+"""
+TensorFlow Specific Parameters
+"""
 class SparkDLTypeConverters(object):
 
     @staticmethod
+    def toTFGraph(value):
+        if isinstance(value, tf.Graph):
+            return value
+        elif isinstance(value, GraphFunction):
+            with IsolatedSession() as issn:
+                issn.importGraphFunction(value, prefix='')
+                g = issn.graph
+            return g
+        else:
+            raise TypeError("Could not convert %s to TensorFlow Graph" % type(value))
+
+    @staticmethod
+    def toTFInputGraph(value):
+        if isinstance(value, TFInputGraph):
+            return value
+        elif isinstance(value, TFInputGraphBuilder):
+            return value
+        elif isinstance(value, tf.Graph):
+            return TFInputGraphBuilder.fromGraph(value)
+        elif isinstance(value, tf.GraphDef):
+            return TFInputGraphBuilder.fromGraphDef(value)
+        else:
+            raise TypeError("Could not convert %s to TFInputGraph" % type(value))
+
+    @staticmethod
+    def asColumnToTensorMap(value):
+        if isinstance(value, dict):
+            strs_pair_seq = [(k, tfx.as_op_name(v)) for k, v in value.items()]
```

Comment:
I think apart from `feetds` and `fetches` in `session.run`, all the references use `tf.Operation` names. I will update the docs.

Reply:
Is it possible though to have two tensors with the same op name in the mappings though?

Reply:
It might happen for ops that return multiple tensors. These operations are not common for inputs and outputs in TensorFlow graphs. I will update the documentation to clarify.

Reply:
Then let's make them tensor names instead of op names (we do that in other transformers in sparkdl anyway). Any reason why op names are much more preferable?

Reply:
We might actually not be able to do this since `TensorFrames` name the output column to the operation name ([we do this in tf_image module](https://github.com/databricks/spark-deep-learning/blob/master/python/sparkdl/transformers/tf_image.py#L140-L144)). 
If there are two tensor outputs from the same operation, there will be a column name conflict.

Reply:
I see. We only output one tensor/op in the image transformers so this is not a problem there. 1/ Could we put a check in the output mappings and warn if there are two tensors with the same op name? 
2/ Let's make sure users only deal with tensor names and op names are internal (not sure if this is already true in the code).